src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -176,7 +186,11 @@ function startWatchingForValueChange(target, targetID) {
   );
 
   Object.defineProperty(activeElement, 'value', newValueProp);
-  activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  if (activeElement.attachEvent) {
+    activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  } else {
+    activeElement.addEventListener('propertychange', handlePropertyChange, false);
```

Comment:
I don't think this will work. According to [MSDN](https://msdn.microsoft.com/en-us/library/ms536956(v=vs.85).aspx), the `propertychange` event is only fired when listened via the `attachEvent` API